Penscratch: Update editor styles

scruffian opened the pull request:
#### Changes proposed in this Pull Request:
Updates editor styles so that they work with Gutenberg 9.9.

It would be better to just use the normal theme styles but the classes in Gutenberg don't match the front end :(

#### Related issue(s):
Part of #3280

scruffian committed:
Penscratch: Update editor styles

jffng reviewed (commented):
Same as #3284, works locally but cannot verify on wpcom.

scruffian committed:
Add all CSS to the editor styles, add a rule to make the post content white, and reference the files locally

scruffian commented:
I updated this to:
- Reference the files with a local reference not a URL
- Queue all CSS in the editor, not just the editor blocks
- Add a rule to make the body of the editor white
- Remove CSS selectors that aren't needed and update old ones to new ones...

MaggieCabrera commented:
Looks like the headings are not working on the editor: 

Frontend | Editor
--- | ---
<img width="485" alt="Screenshot 2021-02-18 at 11 36 14" src="https://user-images.githubusercontent.com/3593343/108344583-98312200-71dd-11eb-98c8-f6b746ea80e9.png"> | <img width="574" alt="Screenshot 2021-02-18 at 11 36 10" src="https://user-images.githubusercontent.com/3593343/108344587-98c9b880-71dd-11eb-8f44-3712a4048359.png">

scruffian committed:
rebase

scruffian committed:
Simplify editor block overrides

scruffian committed:
Simplify editor styles

scruffian commented:
I took a different approach with this. I have removed almost all editor styles, and just kept one for the body content.

This meant that the reset was leaking into the editor and breaking buttons. I have removed the reset from the main stylesheet and queued it separately in the front end.

jffng commented on penscratch-2/css/editor-blocks.css:
```diff
@@ -19,560 +19,6 @@ Description: Used to style Gutenberg Blocks in the editor.
-.editor-default-block-appender textarea.editor-default-block-appender__content {
-	font-family: "Roboto Slab", Georgia, Times, serif;
+body {
+	background: #fff;
```
It's kind of a shame that this line is needed just to override the default background color, otherwise we could get rid of this file entirely.

scruffian replied:
We could add a Gutenberg class to the CSS that targets the site content, but I don't want to start mixing editor CSS into the main file.

jffng commented:
The fonts are loading correctly but I noticed some regressions and styles leaking into the editor UI:

Before | This PR
--------- | ---------
<img width="827" alt="Screen Shot 2021-02-18 at 11 28 04 AM" src="https://user-images.githubusercontent.com/5375500/108388432-a16dbf00-71dc-11eb-8f39-f7ddc1f4ee79.png"> | <img width="1487" alt="Screen Shot 2021-02-18 at 11 29 16 AM" src="https://user-images.githubusercontent.com/5375500/108388451-a763a000-71dc-11eb-9e8d-e778cef9f5e5.png">

scruffian committed:
Remove form styles from the editor

scruffian committed:
Add CSS for the post title

scruffian commented:
Thanks for testing. I moved the form CSS into a separate file, which fixes the problems you found.

MaggieCabrera reviewed (approved):
I think this is ready to ⛵
